qlty-plugins/plugins/scripts/updateLinterVersions.ts

```diff
@@ -47,7 +47,7 @@ const getLintersList = async (): Promise<string[]> => {
           const linterPath = path.join(LINTERS_PATH, dirContent);
           const stat = await fs.promises.stat(linterPath);
           if (stat.isDirectory()) {
-            return linterPath;
+            return dirContent;
```

Comment:
Intentional?

Reply:
Yup, linterPath is the absolute path of the linter directory, `/User/.../qlty/qlty-plugins/plugins/linters/<linter_name>` while dirContent is just the linter name which is desired from this function.